.github/workflows/consuming.yml

```diff
@@ -19,17 +19,10 @@ jobs:
       fail-fast: false
       matrix:
         project: ['admiral', 'lighthouse', 'subctl', 'submariner', 'submariner-operator']
-        deploytool: ['operator', 'helm']
+        deploytool: ['operator']
```

Comment:
Since we don't plan to override this in any permutation, it's better to inline this in step itself (L#65).

As a side note, operator is the default anyhow, so this can be dropped altogether.

Reply:
Done